### PR TITLE
Fixing broken links in documentation

### PR DESCRIPTION
**Pull request by alacuku**
# Description
Fixing broken links in documentation, section "ClusterSharing"
Small update of the tunnelEndpoint-operator documentation

Ref. #230

### CODE REVIEW

**Review by palexster — APPROVED** [review]

File: docs/pages/Architecture/Cluster Sharing/Networking/_index.md [hunk 1]
```diff
@@ -3,12 +3,12 @@ title: Networking
 ---
 
 Liqonet is in charge of connecting networks of different Kubernetes clusters. It is made of three Kubernetes operators:
-* [tunnelEndpointCreator-operator](liqonet_tunEndCreator.md);
-* [route-operator](liqonet_routeOperator.md);
-* [tunnelEndpoint-operator](liqonet_tunnelEndpoint.md).
+* [tunnelEndpointCreator-operator](/architecture/cluster-sharing/networking/liqonet_routeOperator.md);
```
Comment (giorio94) [inline docs/pages/Architecture/Cluster Sharing/Networking/_index.md:6]: The file referenced is different from the one before (liqonet_tunEndCreator). Is this correct?

Comment (alacuku) [reply docs/pages/Architecture/Cluster Sharing/Networking/_index.md:6]: no, it is not. thanks.

**Review by giorio94 — APPROVED** [review]
/lgtm

File: docs/pages/Architecture/Cluster Sharing/Networking/_index.md [hunk 2]
```diff
@@ -3,12 +3,12 @@ title: Networking
...
+* [tunnelEndpointCreator-operator]({{< ref "architecture/cluster-sharing/networking/liqonet_tunEndCreator" >}});
+* [route-operator]({{< ref "architecture/cluster-sharing/networking/liqonet_routeOperator" >}});
+* [tunnelEndpoint-operator]({{< ref "architecture/cluster-sharing/networking/liqonet_tunnelEndpoint" >}}).
 
 The module enables Kubernetes clusters to exchange only the POD traffic, which means that only the POD CIDR subnet of a remote cluster is reachable by a local cluster.
-Thanks to the [resource reflection](../_index.md) pods and nodes of a local cluster can reach also the pods running behind a service in a remote cluster.
+Thanks to the [resource reflection]({{< ref "architecture/cluster-sharing/computing/_index" >}}) pods and nodes of a local cluster can reach also the pods running behind a service in a remote cluster.
 
 ## Architecture
 The following image shows the basic architecture of liqonet.
```
Comment (palexster) [inline docs/pages/Architecture/Cluster Sharing/Networking/_index.md:14]: Liqonet to be changed

**Pull request merged**